static/app/components/onboarding/platformOptionsControl.tsx

```diff
@@ -70,13 +70,21 @@ export type PlatformOptionsControlProps = {
    * Object with default value for each option
    */
   defaultOptions?: Record<string, string[]>;
+  /**
+   * Fired when the value changes
+   */
+  onChange?: () => void;
 };
 
-export function PlatformOptionsControl({platformOptions}: PlatformOptionsControlProps) {
+export function PlatformOptionsControl({
+  platformOptions,
+  onChange,
+}: PlatformOptionsControlProps) {
   const router = useRouter();
   const urlOptionValues = useUrlPlatformOptions(platformOptions);
 
   const handleChange = (key: string, value: string) => {
+    onChange?.();
```

Comment:
Don't you need to pass the new value here?
`docParams`, which is available to the callback, will still hold the old value, right?
Same for product selection.

Reply:
I actually tested the changes in the [PR](https://github.com/getsentry/sentry/pull/77310) and everything was correct

Reply:
But in the implementation of that PR you pass the new values to the callback?
Or am I missing something?

Reply:
or yes...you're right, let me update this code